dietpi/dietpi-software

```diff
@@ -462,6 +464,7 @@ _EOF_
 		EMONHUB=$(cat "$FP_INSTALLED_FILE" | grep -m1 'EMONHUB ' | awk '{print $2}')
 		EMONCMS=$(cat "$FP_INSTALLED_FILE" | grep -m1 'EMONCMS ' | awk '{print $2}')
 		RPIMONITOR=$(cat "$FP_INSTALLED_FILE" | grep -m1 'RPIMONITOR ' | awk '{print $2}')
+		NETDATA=$(sed -n 's/^NETDATA\s*\(.*\)/\1/p' "$FP_INSTALLED_FILE")
```

Comment:
yeah... that's what the "\s*" was supposed to do... but it should be "\s+" 
\s:  whitespace
*:  0 or more occurances
+: 1 or more occurances

I'll fix it